extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -49,6 +49,25 @@
    transition: transform 233ms cubic-bezier(0,0,.21,1);
  }
 
+  header[toolbar] nav[toolbar] {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  header[toolbar] nav[toolbar] ul {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+
+    padding: 0;
+    margin: 0;
+  }
+
+  header[toolbar] nav[toolbar] ul li {
+    margin-bottom: 0;
```

Comment:
This is remove default styles on <li> elements. I was on the fence about this change, and can be removed